exporters/stats/stackdriver/src/test/java/io/opencensus/exporter/stats/stackdriver/StackdriverExportUtilsTest.java

```diff
@@ -208,8 +208,8 @@ public void createDistribution() {
             com.google.api.Distribution.newBuilder()
                 .setMean(2)
                 .setCount(3)
-                .setRange(
-                    com.google.api.Distribution.Range.newBuilder().setMin(0).setMax(5).build())
+                // .setRange(
+                //     com.google.api.Distribution.Range.newBuilder().setMin(0).setMax(5).build())
```

Comment:
It would help to have a TODO here, too, since this is in a different file.

Reply:
Done.